frontend/src/app/modals/add-block-form/add-block-form.component.css

```diff
@@ -108,17 +108,30 @@ td.mat-mdc-cell:first-of-type, td.mat-mdc-footer-cell:first-of-type{
 }
 
 
-#addAsReferenceLabel{
+#addAsReferenceLabel, #blockAddAslabel{
     margin-right: 5px;
     margin-top: 3px
 }
-
+.addReferenceBlockToStepType{
+    margin-left: 10px;
+    display: flex;
+    align-items: center;
+}
+::ng-deep .wayToAddBlockDropDown .mat-mdc-form-field-infix{
```

Comment:
Die css Klasse ist schon in styles vorhanden. Möglicherweise geht es deshalb nicht, die Klasse im Component css zu überschreiben.
<img width="243" alt="image" src="https://github.com/adessoSE/Seed-Test/assets/81535134/274737df-31a8-4672-b73c-41fadc131f0d">